quickstart/201-synapse-secure/main.tf

```diff
@@ -0,0 +1,22 @@
+terraform {
```

Comment:
Please move terraform and provider blocks to provider.tf file.

Reply:
@grayzu done